plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension point="org.eclipse.ui.intro.configExtension">
```

Comment:
Per http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fua_intro_minimal.htm
For an icon add another extension point here like this:

   <extension
         point="org.eclipse.ui.intro">
      <intro
            class="org.eclipse.ui.intro.config.CustomizableIntroPart"
            icon="$nl$/icons/image_obj.gif"
            id="org.eclipse.intro.minimal"
            label="Minimal Intro"/>
   </extension>